src/languages/swift.js

```diff
@@ -473,8 +473,8 @@ export default function(hljs) {
       GENERIC_PARAMETERS,
       ...KEYWORD_MODES,
       {
-        begin: /:/,
-        end: /[{]/,
+        begin: ":",
+        end: "{",
```

Comment:
We prefer regex everywhere we can rather than strings.  My original point was that I think we want `\{` to properly escape `{`, rather than a character group `[{]` which looks a little strange.

Reply:
Ah got it, wasn't aware regex was preferred everywhere!